agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/AiSemanticAttributes.java

```diff
@@ -106,5 +106,16 @@ public final class AiSemanticAttributes {
   public static final AttributeKey<Boolean> IS_PRE_AGGREGATED =
       AttributeKey.booleanKey("applicationinsights.internal.is_pre_aggregated");
 
+  // Azure Function
+
+  public static final AttributeKey<String> INVOCATION_ID = AttributeKey.stringKey("InvocationId");
+  public static final AttributeKey<String> PROCESS_ID = AttributeKey.stringKey("ProcessId");
+  public static final AttributeKey<String> LOG_LEVEL = AttributeKey.stringKey("LogLevel");
+  public static final AttributeKey<String> CATEGORY = AttributeKey.stringKey("Category");
+  public static final AttributeKey<String> HOST_INSTANCE_ID =
+      AttributeKey.stringKey("HostInstanceId");
+  public static final AttributeKey<String> AZ_FUNC_LIVE_LOGS_SESSION_ID =
+      AttributeKey.stringKey("#AzFuncLiveLogsSessionId");
```

Comment:
let's prefix these somehow so it's clear they're scoped to azure functions, maybe something like:

  public static final AttributeKey<String> AZ_FN_INVOCATION_ID = AttributeKey.stringKey("applicationinsights.internal.azurefn.invocation_id");

Reply:
rename constants, but key value needs to match what function team wants though.

Reply:
it's added to attributes directly and we don't need to reprocess it in the LogExporter.